scheduler/feasible.go

```diff
@@ -15,10 +15,13 @@ import (
 )
 
 const (
-	FilterConstraintHostVolumes = "missing compatible host volumes"
-	FilterConstraintCSIVolumes  = "missing CSI plugins"
-	FilterConstraintDrivers     = "missing drivers"
-	FilterConstraintDevices     = "missing devices"
+	FilterConstraintHostVolumes                = "missing compatible host volumes"
+	FilterConstraintCSIPlugins                 = "missing CSI plugins"
+	FilterConstraintCSIVolumesLookupFailed     = "CSI volume lookup failed"
+	FilterConstraintCSIVolumeNotFoundTemplate  = "missing CSI Volume %s"
```

Comment:
I actually introduced these constants because I thought they would be useful for matching the filtered reason to block & retry evals that failed for transient reasons. It turns out it's not useful; that logic doesn't have to rely on matching filtered causes. If the constants seem like they clean things up we can keep them, but we could also just drop the strings in line. I don't think any of them are used frequently.

Reply:
I think keeping them is probably fine for now, but might be worth removing in the future